src/tests/testthat/testProgramData.R

```diff
@@ -14,13 +14,19 @@ testthat::test_that("warning is shown if incomplete program data is present", {
 
 testthat::test_that("warning is not shown if complete program data is present", {
 
-    uploadSpectrumFileAndSwitchTab("Review input data")
+    uploadSpectrumFileAndSwitchTab("Upload programmatic data")
```

Comment:
Previously this test relied on there being built in data for Malawi. Now we need to provide it explicitly.